### PR TITLE
Buy button redo

### PR DESCRIPTION
**Pull request by frankiebee**
Redo the buy view in vlads design and some maintenance and clean up
as well as fix a bug where if you click the buy button in conf-tx it tries to send the tx
screen shot 2017-04-24 at 4 11 27 pm
screen shot 2017-04-24 at 4 11 33 pm

### CODE REVIEW

**Comment by kumavis** [comment]
image

File: ui/app/components/buy-button-subview.js [hunk 1]
```diff
@@ -46,58 +53,79 @@ BuyButtonSubview.prototype.render = function () {
...
+        style: {
+          position: 'relative',
+          right: '35px',
+          marginTop: '20px',
+          marginBottom: '20px',
+        },
+      }, [
+        h(RadioList, {
+          defaultFocus: props.buyView.subview,
+          lables: [
```
Comment (flyswatter) [inline ui/app/components/buy-button-subview.js]: `labels`.

**Review by danfinlay — CHANGES_REQUESTED** [review]
Spelling fix

**Review by danfinlay — APPROVED** [review]
LGTM

**Pull request merged**